### PR TITLE
BGDIINF_SB-2423: app_version

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,9 +14,9 @@ ENV GROUP geodata
 RUN apt-get update -qq \
     && DEBIAN_FRONTEND=noninteractive apt-get install -qq -y --upgrade ca-certificates \
     && DEBIAN_FRONTEND=noninteractive apt-get install -qq -y  \
-        libgeos-dev \
-        gettext-base \
-        apache2 libapache2-mod-wsgi-py3 \
+    libgeos-dev \
```
Comment (ltshb) [inline Dockerfile:17]: Oops some unwanted whitespaces changes, probably done via vscode, see https://github.com/geoadmin/tool-dev-setup/pull/114

**Review by ltshb — CHANGES_REQUESTED** [review]
Good just revert the white spaces in dockerfile. Maybe `base_version.ini` is more appropriate ?

**Review by procrastinatio — APPROVED** [review]
OK, but I'm not sure it has any real signification as the docker images have all kinds of tags.

**Review by hansmannj — APPROVED** [review]
:+1: Thanks!

File: Dockerfile (hunk 1 above)
Comment (rebert) [reply Dockerfile]: as the variable is called app_version all over, I would like to keep it app_version

**Review by ltshb — APPROVED** [review]
👍🏼

**Pull request merged**